packages/AI/VectorDupe/src/duplicateRecordDetector.ts

```diff
@@ -151,7 +163,7 @@ export class DuplicateRecordDetector extends VectorBase {
         return response;
     }
 
-    private async GetRecordsByEntityID(entityID: number, recordIDs: PrimaryKeyValueBase[]): Promise<BaseEntity[]> {
+    private async GetRecordsByEntityID(entityID: number, recordIDs: CompositeKey[]): Promise<BaseEntity[]> {
         const rvResult = await this._runView.RunView({
```

Comment:
@JS-BC  why are you getting metadata this way? Remember, use the Metadata object and go back through ALL your code and check for this now, very inefficient and unnecessary.